refocus-bdk-server.js

```diff
@@ -486,42 +486,41 @@ module.exports = (config) => {
   } // heartBeat
 
   /**
-   * Gets room type information
-   * @param {*} roomTypeId;
-   * @returns {object} Roomtype information or null
+   * Gets room payload
+   * @param {*} roomId;
+   * @returns {object} Room information or null
    */
-  async function getRoomTypeById(roomTypeId) {
+  async function getRoomById(roomId) {
     try {
-      if (roomTypeId) {
+      if (roomId) {
         // eslint-disable-next-line max-len
-        const response = await generic.get(`${SERVER}${API}${ROOM_TYPES_ROUTE}/${roomTypeId}`,
+        const response = await generic.get(`${SERVER}${API}${ROOMS_ROUTE}/${roomId}`,
           TOKEN, DEFAULT_TRIES, null, PROXY_URL);
         return response.body;
       }
     } catch (error) {
-      log.error('failed return Roomtype data from Refocus:', error.message);
+      log.error('failed return Room data from Refocus:', error.message);
       return null;
     }
     return null;
   }
 
   /**
-   * Checks RoomType bot list for bot triggering event
-   * @param {Object} event - refocusEvent
+   * Checks Room by id for bot triggering event
+   * @param {Object} roomId - refocusEvent
    * @param {String} bot - botName
    * @returns {boolean}
    */
-  async function isBotInstalledInRoom(event, bot) {
-    const eventRoomTypeId = event.Room.RoomType.id;
+  async function isBotInstalledInRoom(roomId, bot) {
     try {
-      const botsInRoomType = await getRoomTypeById(eventRoomTypeId);
-      if (botsInRoomType) {
-        const botsInstalled = botsInRoomType.bots.find((b) => b === bot);
+      const botsInRoom = await getRoomById(roomId);
+      if (botsInRoom) {
+        const botsInstalled = botsInRoom.bots.find((b) => b === bot);
```

Comment:
"The find() method returns the value of the first element in the provided array that satisfies the provided testing function."
How about var name `installedBot` or `botInstalled,` as singleton?

Reply:
Alternatively, you could also do
`const botsInstalled = room.bots`   (assuming we agree to rename the 'botsInRoom' to 'room')
`return botsInstalled.includes(bot)`

Reply:
Cheers, I'm going with the second approach you suggested, makes the reading clearer that we want to know if bot x is installed in that room.